lms/static/sass/course/modules/_calculator.scss

```diff
@@ -62,6 +68,12 @@ div.calc-main {
       @include box-sizing(border-box);
       padding: lh();
 
+      label.sr {
```

Comment:
Hmm, I tried looking in the page's DOM for where this is needed and didn't see any rules. Should we just remove this?